CPAC/resources/configs/pipeline_config_default.yml

```diff
@@ -409,7 +409,7 @@ anatomical_preproc:
       monkey: False
 
     FSL-BET:
-
+      Robustfov : Off
```

Comment:
Can you add a comment above this line in the default pipeline to explain what the option is and how to set it? The comment should propogate automatically to the other pipelines that include the key.

Reply:
Thanks. Added now.